DataImport.Models/DataMapSerializer.cs

```diff
@@ -125,6 +129,12 @@ private JObject SerializeObject(IReadOnlyList<ResourceMetadata> nodeMetadatas, I
             return result;
         }
 
+        private JObject SerializeObjectForDeleteById(DataMapper node)
+        {
+            var result = new JObject { new JProperty("Id", new JObject { new JProperty("Column", node.SourceColumn) }) };
+            return result;
+        }
+
         private JArray SerializeArray(ResourceMetadata arrayItemMetadata, IReadOnlyList<DataMapper> nodes)
```

Comment:
@simpat-adam 
Can we have separate DataMapSerializer class for just for Delete operations ? This will make sure to have single response across the classes. If we add delete by natural key we can plugin one more class or over loads on this class !